### PR TITLE
Add "view" command and relevant tests

### LINKED ISSUE #66: Add ViewCommand

### PR DESCRIPTION
**Pull request by LemonDrew**
Closes #66

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/ViewStatusCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import seedu.address.model.person.Name;
+
+
+
+public class ViewStatusCommandTest {
+
+    @Test
+    public void constructor_validInput_success() {
+        Name name = new Name("John Doe");
+        String job = "Software Engineer";
```
Comment (SQ77) [inline src/test/java/seedu/address/logic/commands/ViewStatusCommandTest.java:20]: Can this be changed to `Job job` now that the Job class has been implemented?

**Review by SQ77 — COMMENTED** [review]
Some changes can be made with the implementation of the Job class in #71

File: src/main/java/seedu/address/logic/commands/ViewStatusCommand.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        this.name = name;
+        this.job = job;
+    }
+
+    /**
+     * Throws a simple exception
+     */
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        throw new CommandException("Remark command not implemented yet");
```
Comment (SQ77) [inline src/main/java/seedu/address/logic/commands/ViewStatusCommand.java:43]: Should this be "View command" instead of "Remark command"?

**Review by SQ77 — COMMENTED** [review]
Very minor typo to change, otherwise LGTM.

**Review by bigismols — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/ViewStatusCommand.java (hunk 2 above)
Comment (bigismols) [reply src/main/java/seedu/address/logic/commands/ViewStatusCommand.java:43]: Will be fixed in subsequent PRs

**Pull request merged**